smart_card_connector_app/src/_locales/en/messages.json

```diff
@@ -31,6 +31,22 @@
     "message": "Smart Card readers",
     "description": "The header displayed in front of the readers list"
   },
+  "initReaderTooltip": {
+    "message": "Initializing...",
+    "description": "Tooltip indicating that the reader is being initialized"
+  },
+  "successReaderTooltip": {
+    "message": "The reader is initialized.",
+    "description": "Tooltip indicating that the reader is initialized successfully"
+  },
+  "failureReaderTooltip": {
+    "message": "Failed to connect reader.",
```

Comment:
nit: "the reader"

Reply:
It's grammatically correct and it's shorter so I thought it would be better.

Reply:
Is it correct?
I'm not a native speaker, but I would think that as we're showing this text for a *specific* reader, then we have to use "the" with it. Also I checked a couple of webpages regarding English Grammar and couldn't find why this sentence would be a valid case when no article can be used:
* https://7esl.com/use-no-article-rules/
* https://www.englishgrammar.org/omission-articles/

Reply:
On grammarly.com when writing "Failed to connect device" it gives a grammatical error, but when writing "Error: Failed to connect device" it doesn't, so I'm assuming there are exceptions, but other than that it comes from intuition. Nonetheless, I'll change it.